### ISSUE TITLE
Headwind doesn't work: rustywind syntax error

### ISSUE THREAD

**Issue by thiras** [heybourn/headwind#113]
**Describe the bug**
Headwind doesn't work. It throws an error when I run `Sort TailwindCSS Classes on Entire Workspace`;

```
Headwind error: /home/thiras/.vscode/extensions/heybourn.headwind-1.7.0/node_modules/rustywind/bin/rustywind: 1: Syntax error: ")" unexpected
```

**To Reproduce**
1. Go to install the extension
2. Restart VSCode
3. Run `Sort TailwindCSS Classes on Entire Workspace`

**Expected behavior**
The extension should fix the entire workspace

**Screenshots**
![Screenshot from 2021-03-11 16-46-43](https://user-images.githubusercontent.com/5150432/110796774-644f9680-8289-11eb-9b48-b967411b2696.png)

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser Firefox
 - Version 86

**Smartphone (please complete the following information):**

**Additional context**
```
Version: 1.54.1
Commit: f30a9b73e8ffc278e71575118b6bf568f04587c8
Date: 2021-03-04T22:38:50.094Z
Electron: 11.3.0
Chrome: 87.0.4280.141
Node.js: 12.18.3
V8: 8.7.220.31-electron.0
OS: Linux x64 5.8.0-44-generic
```

**Comment by praveenperera** [comment]
Hey @thiras do any Headwind configuration in vscode settings.json?

**Comment by thiras** [comment]
@praveenperera nope. I didn't change anything on settings. It's all default. No custom config on settings.json

**Comment by praveenperera** [comment]
I am unable to recreate this issue

**Comment by timsun28** [comment]
Hi, 
I got the same issue as mentioned above: 
Headwind error: /home/timo/.vscode/extensions/heybourn.headwind-1.7.0/node_modules/rustywind/bin/rustywind: 1: Syntax error: ")" unexpected

I just installed the extension and did not make any modifications. The plugin works well when saving files, so it only gives me this error when running the command to style the entire workspace. 

I'm also having this issue on Ubuntu 20.04 with version 1.7.0 of Headwind. 

Please let me know if I can supply you with more information needed to discover what is going on here.

**Comment by mattrossman** [comment]
Same error here, I'm on Ubuntu 20.04 via WSL2.

Headwind: v1.7.0
VSCode: 1.65.1

**Comment by EmilioBravo** [comment]
Same error here
OS: Ubuntu 22.04
Headwind: v1.7.0
VSCode: 1.67.2

**Comment by kissu** [comment]
Same error:
- basic ESlint + Prettier configuration
- using Tailwind
- works perfectly on a component individually
- used for a Nuxt.js app
- Ubuntu 20.04
- VScode 1.69.1
- Headwind 1.7.0
- I'm using `"headwind.runOnSave": true`
- do also have `"files.autoSave": "onFocusChange"`

I'm not sure how we can debug this one. The Electron devtools are not more verbose + I don't see a `Headwind (Extension)` in the output.

I have a quite big configuration, so I'll maybe try to replicate it into a vanilla Ubuntu VM.
Then into an Arch Linux VM if I get some time. This is maybe coming from the distribution?

**Comment by ran-dall** [comment]
Same error.

- Gentoo
- Headwind 1.7.0
- VSCode 1.70.1

**Comment by ryantaussig** [comment]
I'm experiencing the same issue.

- Ubuntu 22.04
- VSCode 1.73.0
- Headwind 1.7.0

**Comment by dbm03** [comment]
Same issue.
- Ubuntu 22.10
- VSCode 1.74.2
- Headwind 1.7.0

**Comment by AkioCode** [comment]
Same issue.

- Ubuntu 22.04
- VSCode 1.77.3
- Headwind 1.7.0

**Comment by MateusJSouza** [comment]
Same issue.

- Ubuntu 22.04
- VSCode 1.84.2
- Headwind 1.7.0
